puppet/modules/web/manifests/jenkins.pp

```diff
@@ -1,25 +1,17 @@
-class web::jenkins(
+class web::jenkins (
   Stdlib::Fqdn $hostname = 'ci.theforeman.org',
-  Stdlib::Absolutepath $webroot = '/var/www/vhosts/jenkins/htdocs',
   Boolean $https = false,
 ) {
   include web::base
 
-  $proxy_pass = {
-    'path'          => '/',
-    'url'           => 'http://localhost:8080/',
-    'keywords'      => ['nocanon'],
-    'no_proxy_uris' => ['/.well-known'],
-  }
-
-  if $https {
-    include web::letsencrypt
-
-    letsencrypt::certonly { $hostname:
-      plugin        => 'webroot',
-      domains       => [$hostname],
-      webroot_paths => [$webroot],
-    }
+  $proxy_attrs = {
+    'allow_encoded_slashes' => 'nodecode',
+    'proxy_pass' = {
+      'path'          => '/',
+      'url'           => 'http://localhost:8080/',
+      'keywords'      => ['nocanon'],
+      'no_proxy_uris' => ['/.well-known'],
+    },
```

Comment:
```
Could not parse for environment *root*: Syntax error at ','
```

Reply:
I still miss something in my vim config from when I moved to my new laptop.